### DISCUSSION TITLE
Add custom number to path for sorting

### DISCUSSION THREAD

**Discussion by dogo77** [beetbox/beets#5166] [Q&A] [answered]
I have some trouble to sort a couple of Audio Books.

Audio Books relevant part of the config:
```
"albumtype:Audiobook": Audiobooks/$albumartist/[$year] $album%aunique{} [$format]/%if{$multidisc,disc $disc }%if{$disctitle,$disctitle}/$track $title
```

I have all Harry Potter Audio Books nicely sorted in 
```
/Archive/Audiobooks/J. K. Rowling/[2001] Harry Potter und der Gefangene von Askaban [OGG]
/Archive/Audiobooks/J. K. Rowling/[2001] Harry Potter und der Stein der Weisen [MP3]
/Archive/Audiobooks/J. K. Rowling/[2007] Harry Potter und der Halbblutprinz [MP3]
/Archive/Audiobooks/J. K. Rowling/[2007] Harry Potter und die Kammer des Schreckens [OGG]
/Archive/Audiobooks/J. K. Rowling/[2009] Harry Potter und die Heiligtümer des Todes [MP3]
/Archive/Audiobooks/J. K. Rowling/[2009] Harry Potter und der Feuerkelch [MP3]
/Archive/Audiobooks/J. K. Rowling/[2010] Harry Potter und der Orden des Phönix [MP3]
```
what i'd **like** to achieve is

```
/Archive/Audiobooks/J. K. Rowling/[1] [2001] Harry Potter und der Stein der Weisen [MP3]
/Archive/Audiobooks/J. K. Rowling/[2] [2007] Harry Potter und die Kammer des Schreckens [OGG]
/Archive/Audiobooks/J. K. Rowling/[3] [2001] Harry Potter und der Gefangene von Askaban [OGG]
/Archive/Audiobooks/J. K. Rowling/[4] [2009] Harry Potter und der Feuerkelch [MP3]
/Archive/Audiobooks/J. K. Rowling/[5] [2010] Harry Potter und der Orden des Phönix [MP3]
/Archive/Audiobooks/J. K. Rowling/[6] [2007] Harry Potter und der Halbblutprinz [MP3]
/Archive/Audiobooks/J. K. Rowling/[7] [2009] Harry Potter und die Heiligtümer des Todes [MP3]
```
read: let the chronological order be reflected in the file path.
I couldn't find anything in the tags that would help.
I guess i have to add something but before going try and error i'd like to ask if anybody encountered a similar situation and has an receipt. :))

ty

**Comment by benpage2** [comment]
You could add a custom tag such using `beet modify [query] release_order=[some number]` for every book and then add that number in your audiobook path format.

Alternatively, setting `original_date: yes` in your configuration may give years in chronological order? Although I'm not 100% sure if that applies to the Harry Potter series.

**Reply by dogo77** [reply]
got no notification about that answer, so as an late reply:

will try that, TY! :)

**Reply by dogo77** [reply, marked as answer]
> You could add a custom tag such using `beet modify [query] release_order=[some number]` for every book and then add that number in your audiobook path format.

This works like a charm.

`paths:
    "album:'Harry Potter'": Hörspiele/$albumartist/%ifdef{release_order} $album%aunique{} [$year] [$format]/%if{$multidisc,disc $disc }%if{$disctitle,$disctitle}/$track $title`


`beet modify 'Harry Potter und der Stein der Weisen' release_order='[1]'` 




Just be sure to have that  `"album:'Harry Potter'":` part **before** your path definitions of Audiobooks or it won't work ;)

thanks a lot @benpage2 !!
